### ISSUE TITLE
Invalid bounding box on Font/Text / LineSpacing?

### ISSUE THREAD

**Issue by mortoray** [fusetools/fuselibs-public#243]
We don't appear to be calculating the bounding box of text correctly. Perhaps this is an issue with the negative LineSpacing property.

https://www.fusetools.com/community/forums/bug_reports/typography_issues

**Comment by yupferris** [comment]
negative LineSpacing?

**Comment by mortoray** [comment]
Yes, this squishes the lines closer together in applications that support it. I'd assume we could support it as well, but perhaps we just didn't test it and adjust our algorithms.
